### DISCUSSION TITLE
Unexpected number of forces and stress read from the training file with MACE 0.3.12

### DISCUSSION THREAD

**Discussion by riyazpu** [ACEsuit/mace#888] [General]
"2025-03-26 17:39:16.765 INFO: ===========LOADING INPUT DATA===========
2025-03-26 17:39:16.765 INFO: Using heads: ['default']
2025-03-26 17:39:16.765 INFO: =============    Processing head default     ===========
2025-03-26 17:39:19.497 INFO: Training file 1/1 [3010 configs, 3010 energy, 3010 forces, 3010 stresses] loaded from 'train.xyz'
2025-03-26 17:39:19.498 INFO: Total training set [3010 configs, 3010 energy, 3010 forces, 3010 stresses]
2025-03-26 17:39:19.500 INFO: Using random 5% of training set for validation with indices saved in: ./valid_indices_123.txt
2025-03-26 17:39:19.500 INFO: Validation set contains 150 configurations [150 energy, 150 forces, 150 stresses]
2025-03-26 17:39:20.030 INFO: Test file 1/1 [607 configs, 607 energy, 607 forces, 607 stresses] loaded from 'test.xyz'
2025-03-26 17:39:20.031 INFO: Total test set (607 configs):
2025-03-26 17:39:20.031 INFO: Default_default: 607 configs, 607 energy, 607 forces, 607 stresses
2025-03-26 17:39:20.031 INFO: Total number of configurations: train=2860, valid=150, tests=[Default_default: 607]," 

Generated with MACE 0.3.12, the number of forces and stress should be much higher.  But MACE 0.3.10 correctly generates this 

"2025-03-16 23:09:58.681 INFO: Using heads: ['default']
2025-03-16 23:09:58.681 INFO: =============    Processing head default     ===========
2025-03-16 23:10:01.499 INFO: Training set [3010 configs, 3010 energy, 1158096 forces] loaded from 'train.xyz'
2025-03-16 23:10:01.509 INFO: Using random 5% of training set for validation with indices saved in: ./valid_indices_123.txt
2025-03-16 23:10:01.510 INFO: Validaton set contains 150 configurations [150 energy, 57930 forces]
2025-03-16 23:10:02.114 INFO: Test set (607 configs) loaded from 'test.xyz':
2025-03-16 23:10:02.114 INFO: Default_default: 607 configs, 607 energy, 210561 forces
2025-03-16 23:10:02.115 INFO: Total number of configurations: train=2860, valid=150, tests=[Default_default: 607],
2025-03-16 23:10:02.115 INFO: ==================Using multiheads finetuning mode=================="  .

**Comment by ilyes319** [comment]
Switched to showing the number of configurations with forces instead of total number of forces, as this is not super telling.
